src/network/networkd-network-gperf.gperf

```diff
@@ -140,6 +140,8 @@ Network.PrimarySlave,                        config_parse_bool,
 Network.IPv4ProxyARP,                        config_parse_tristate,                                    0,                             offsetof(Network, proxy_arp)
 Network.ProxyARP,                            config_parse_tristate,                                    0,                             offsetof(Network, proxy_arp)
 Network.IPv6ProxyNDPAddress,                 config_parse_ipv6_proxy_ndp_address,                      0,                             0
+Network.IPv4ReversePathFilter,               config_parse_int,                                         0,                             offsetof(Network, ipv4_rp_filter)
+Network.IPv6ReversePathFilter,               config_parse_int,                                         0,                             offsetof(Network, ipv6_rp_filter)
```

Comment:
Please introduce a specific parser for the settings, that should be take one of 'no' (or boolean false), ’strict', or 'loose'.

Reply:
yes right